tests/blend.py

```diff
@@ -257,86 +254,32 @@ def test_multi_channel(channel_check, channel_check_inverse, range_all,
     np.testing.assert_array_equal(expected, result)
 
 
-def test_channel_size_mismatch(range_all, color_white):
+def test_linear_rgb_size_mismatch(range_all, color_white):
```

Comment:
This should be an specific check in the code, which will raise a different error, but just check for `ValueError` with no string match.

Reply:
done in `refactor names and revert to version 0.0.1`

Reply:
https://github.com/sorgerlab/minerva-lib-python/pull/17/commits/29ed61d369e2d506ea15686cd45fd49e12580ee1#diff-4438daa1ceea9cdeeef65ab8cfb5075dR275